graph/src/data/subgraph/schema.rs

```diff
@@ -66,7 +66,10 @@ impl SubgraphManifest {
         let mut entity = HashMap::new();
         entity.insert("id".to_owned(), id.clone().into());
         entity.insert("specVersion".to_owned(), self.spec_version.into());
-        entity.insert("description".to_owned(), self.description.into());
+        let description = self.description.unwrap_or_default();
+        entity.insert("description".to_owned(), description.into());
```

Comment:
This will set the description attribute to an empty string, right? We can work with that but it would be cleaner to set it to `Value::Null` instead. Same with the other optional fields.

Reply:
That's much better.